tasks/css.js

```diff
@@ -117,7 +118,7 @@ gulp.task('css:bundle', () => gulp.src(`${pathBuilder.scssSrcDir}/**/*.scss`)
         postcss([
             // Converts any specified assets to data URIs
             assets({
-                loadPaths: [pathBuilder.imgSrcDir, pathBuilder.importedAssetsDistDir]
+                loadPaths: [pathBuilder.imgSrcDir, config.assetDistDir, path.dirname(config.assetDistDir)]
```

Comment:
Do you need both `config.assetDistDir` and `path.dirname(config.assetDistDir)` here?